next.config.mjs

```diff
@@ -1,4 +1,16 @@
 /** @type {import('next').NextConfig} */
-const nextConfig = {};
-
-export default nextConfig;
+const nextConfig = {
+    images: {
+      remotePatterns: [
+        {
+          protocol: 'https',
+          hostname: 'de-test-bucket-8285.s3.ap-south-1.amazonaws.com',
```

Comment:
This is not the best place to store Bucket URL, Please store this in environment variables

Reply:
this change is done by Akhil sir

Reply:
Okay will discuss with him